lib/composer_element_template.ex

```diff
@@ -0,0 +1,127 @@
+
+
+defmodule SimpleComposerDemo.Elements.Composer do
```

Comment:
I would keep `Membrane.VideoCompositor` as the module's name (also `membrane_video_compositor` as a filename). It is not a template; it is a rather POC of Compositor which will add features later on.